examples/analytics.amp.html

```diff
@@ -127,6 +127,22 @@
 </script>
 </amp-analytics>
 
+<!-- Krux tracking. Ensure you replace KMnP7vhh with your configuration ID. -->
+<amp-analytics type="krux" config="https://cdn.krxd.net/controltag/amp/KMnP7vhh.json">
```

Comment:
Yes, the production change to the service hasn't been pushed yet.  Will do that Monday morning.